testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/OfflineTokenTest.java

```diff
@@ -832,6 +836,7 @@ private void conductOfflineTokenRequest(String expectedRefreshAlg, String expect
             TokenSignatureUtil.changeRealmTokenSignatureProvider(adminClient, expectedIdTokenAlg);
             TokenSignatureUtil.changeClientAccessTokenSignatureProvider(ApiUtil.findClientByClientId(adminClient.realm("test"), "offline-client"), expectedAccessAlg);
             offlineTokenRequest(expectedRefreshAlg, expectedAccessAlg, expectedIdTokenAlg);
+            offlineTokenRequestWithScopeParameter(expectedRefreshAlg, expectedAccessAlg, expectedIdTokenAlg);
```

Comment:
@mposolda this is my addition. Do you want to keep it?
If we do not want it, _offlineTokenRequestWithScopeParameter_ method and changes in _testRefreshWithOfflineToken_ method will be removed.

Reply:
YEah, looks good. Thanks!